src/containers/audio-rewards-page/WalletModal.tsx

```diff
@@ -279,6 +303,10 @@ const shouldAllowDismiss = (modalState: Nullable<ModalState>) => {
     !(
       modalState.stage === 'CONNECT_WALLETS' &&
       modalState.flowState.stage === 'REMOVE_WALLET'
+    ) &&
+    !(
+      modalState.stage === 'SEND' &&
+      modalState.flowState.stage === 'AWAITING_CONVERTING_ETH_AUDIO_TO_SOL'
```

Comment:
Comment a few lines above `// Allow dismiss on every stage except claiming` has been lying now for a while